src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,41 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
         --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
+        --admin-user-enabled -a  : Whether the admin user account is enabled.  Allowed values: false,
```

Comment:
@derekbekoe for items which are exposed as flags on create, they are exposed with true/false or enabled/disabled in the updates, otherwise, as @skkestrel points out, you can't disable them. There are several places in the CLI, particularly network (and webapp I believe), that follow this pattern.